src/marqo/marqo_cloud_instance_mappings.py

```diff
@@ -56,6 +63,7 @@ def _refresh_urls(self, timeout=None):
 
         if not response.ok:
             mq_logger.warning(response.text)
+            return None
```

Comment:
Since this is something we missed before, did we add the missing test?

Reply:
Good point. Added relevant test: https://github.com/marqo-ai/py-marqo/blob/pandu/no-warnings-on-index-instantiation/tests/v0_tests/test_marqo_cloud_instance_mapping.py#L174